params/protocol_params.go

```diff
@@ -138,6 +138,8 @@ const (
 
 	// The Refund Quotient is the cap on how much of the used gas can be refunded
 	RefundQuotient uint64 = 5
+
+	MaxAddressGrindAttempts int = 1000 // Maximum number of attempts to grind an address to a valid one
```

Comment:
I think this should be a factor of the block gas limit (how many iterations given 5m gas or whatever the limit is?)

Reply:
Good idea, would be interested to hear @kiltsonfire and @gameofpointers thoughts

Reply:
Since this is for people who are willing to pay for the compute and not for people who will do the work outside, idk if it should be a factor on gas limit. This is a static cost and there is an expectation around it. We should see what the expectation to find a valid address is and just pick a constant that is 3x the expectation.

Reply:
@gameofpointers I think you should be able to pay for compute up to the gas limit. After the gas limit, there is no reason to continue computing so end there. Since 1000 attempts will cross over the gas limit it's probably too high but I can't think of a better number.